### ISSUE TITLE
Cannot find module '/app/node_modules/react-icons/lib'. Please verify that the package.json has a valid "main" entry

### ISSUE THREAD

**Issue by vallerion** [react-icons/react-icons#248]
**Describe the problems**
hi guys, i use react + next.js@9.0.2, react-icons@3.7.0, after update next 7 -> 9 version i got error while build:
`Error: Cannot find module '/app/node_modules/react-icons/lib'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:228:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:365:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:612:27)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:683:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.react-icons (/app/.next/server/static/development/pages/question/public.js:7954:18)
    at __webpack_require__ (/app/.next/server/static/development/pages/question/public.js:23:31)
    at Module../components/header/navbar/Toggler.js (/app/.next/server/static/development/pages/question/public.js:3188:69)
    at __webpack_require__ (/app/.next/server/static/development/pages/question/public.js:23:31)
    at Module../components/header/Header.js (/app/.next/server/static/development/pages/question/public.js:2766:73)
    at __webpack_require__ (/app/.next/server/static/development/pages/question/public.js:23:31)
    at Module../components/header/HeaderContainer.js (/app/.next/server/static/development/pages/question/public.js:2831:65)
    at __webpack_require__ (/app/.next/server/static/development/pages/question/public.js:23:31)
    at Module../components/layout/SitePage.js (/app/.next/server/static/development/pages/question/public.js:3822:82)
    at __webpack_require__ (/app/.next/server/static/development/pages/question/public.js:23:31) {
  code: 'MODULE_NOT_FOUND',
  path: '/app/node_modules/react-icons/package.json',
  requestPath: 'react-icons'`

**my code using react-icons:**
```jsx

import {
  FaFacebookF, FaLinkedinIn, FaTelegramPlane, FaVk
} from 'react-icons/fa'

...

              <a href="#">
                <FaVk />
              </a>
...
```

**Comment by novonimo** [comment]
Make sure you are importing correct icon name  from "react-icon"

It looks like you imported correct icons in this code snippet but checks your all importing expression.

**Comment by flexlee** [comment]
Did you use <IconContext> to wrap around the icon? I had similar problem but I forgot the <div> between the <IconContect.Provider> and the <Icon>, like in the doc. Having the <div> solved the problem.

**Comment by kamijin-fanta** [comment]
resolved in #273
